dockerfiles/Dockerfile.build

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:11
+LABEL maintainer="Digirati <opensource@digirati.com>"
+
+USER root
```

Comment:
I don't think we need to switch to `root` here - the user defaults to `root` for the `openjdk:11` image.